diagnostic.md

```diff
@@ -148,9 +167,9 @@ div {
   clear: left;
 }
 ```
-
+div
 <!-- your answer starts here -->
-
+divFour will be on the same line as divOne, with Divfour appearing first if read from right to left.
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short:
```
2 3      1
4 5
```